geode-core/src/main/java/org/apache/geode/management/internal/configuration/realizers/GatewayReceiverRealizer.java

```diff
@@ -89,8 +90,19 @@ public boolean exists(GatewayReceiverConfig config, InternalCache cache) {
   }
 
   @Override
-  public RuntimeInfo get(GatewayReceiverConfig config, InternalCache cache) {
-    return null;
+  public GatewayReceiverInfo get(GatewayReceiverConfig config, InternalCache cache) {
+    if (cache.getGatewayReceivers() == null || cache.getGatewayReceivers().size() == 0) {
+      return null;
+    }
+
+    GatewayReceiver receiver = cache.getGatewayReceivers().iterator().next();
+
+    GatewayReceiverInfo info = new GatewayReceiverInfo();
+    info.setBindAddress(receiver.getBindAddress());
+    info.setHostnameForSenders(receiver.getHostnameForSenders());
+    info.setPort(receiver.getPort());
+    info.setRunning(receiver.isRunning());
+    return info;
```

Comment:
how will this code handle multiple gateway receivers?  it seems to operate only on the first one.

Reply:
Even when the API might suggest otherwise (`getGatewayReceivers()`), there can only be one `gateway-receiver` per member. There's some information about this in [GEODE-3987](https://issues.apache.org/jira/browse/GEODE-3987).